root/static/scripts/tests/release-editor/fields.js

```diff
@@ -183,3 +183,30 @@ fieldTest("tracks are set correctly when the cdtoc is changed", function (t, rel
     );
     t.ok(_.last(medium.tracks()).isDataTrack());
 });
+
+fieldTest("Tracks' time are changed correctly when inputting values in the medium tracklist editing form tab", function (t, release){
+    t.plan(8);
+
+    var medium = new fields.Medium({ tracks: [ {} ] }, release);
+    
+    var lengths = [
+        {5},
```

Comment:
`{...}` notation is for object and should contain `key: value` list. I suggest removing curly brackets.

Reply:
Please **replace** curly brackets with quotes (for both `lengths` and `output` arrays elements. Sorry if it was unclear.

Reply:
Also that was not about the `tracks` array, https://github.com/metabrainz/musicbrainz-server/pull/750/commits/4676bcda180a5cda926a84958e34b0fb2e8a8268 should probably be reverted.